RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate ShareView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Try to share something with the Rocket Chat App.
- Test sharing only one file and with multiple files too
- Also you should be able to share files with your previous conversations
- Check if the thumb attachment and preview attachment  are showing correctly
- Furthermore, check if the Attachment View to see images inside the normal app is working properly

## Screenshots

## Types of changes




## Checklist



## Further comments
